driver/razerkbd_driver.h

```diff
@@ -120,7 +122,7 @@
 
 #define RAZER_BLACKWIDOW_CHROMA_WAIT_MS 1
 #define RAZER_BLACKWIDOW_CHROMA_WAIT_MIN_US 600
-#define RAZER_BLACKWIDOW_CHROMA_WAIT_MAX_US 800
+#define RAZER_BLACKWIDOW_CHROMA_WAIT_MAX_US 5000
```

Comment:
You should introduce new values for your device, see razermouse_driver.c how it's done there

Also these values don't make much sense, it's a usleep that's actually sleeping in a more or less random value between these two numbers, so every time it's sleeping some time between 600 and 5000 nano(?)seconds; so min & max are best kept 100 apart

Reply:
Done